modules/backing-up-red-hat-quay.adoc

```diff
@@ -0,0 +1,320 @@
+[[backing-up-red-hat-quay]]
+== Backing up Red Hat Quay
+
+This procedure is exclusively for OpenShift Container Platform and NooBaa deployments.
+
+.Prerequisites
+
+* A {productname} deployment on OpenShift Container Platform.
+
+
+.Procedure
+
+
+. Backup the `QuayRegistry` custom resource by exporting it:
++
+[source,terminal]
+----
+$ oc get quayregistry <quay-registry-name> -n <quay-registry-namespace> -o yaml > quay-registry.yaml
+----
+
+. Edit the resulting `quayregistry.yaml` and remove the status section and the following metadata fields:
++
+[source,yaml]
+----
+  metadata.creationTimestamp
+  metadata.finalizers
+  metadata.generation
+  metadata.resourceVersion
+  metadata.uid
+----
+
+. Backup the managed keys secret:
++
+[NOTE]
+====
+If you are running a version older than Red Hat Quay 3.7.0, this step can be skipped. Some secrets are automatically generated while deploying Quay for the first time. These are stored in a secret called `<quayname>quay-registry-managed-secret-keys` in the QuayRegistry namespace.
+====
++
+[source,terminal]
+----
+$ oc get secret -n <quay-namespace> <quay-registry-name>-quay-registry-managed-secret-keys -o yaml > managed-secret-keys.yaml
+----
+
+. Edit the the resulting `managed-secret-keys.yaml` file and remove all owner references. Your `managed-secret-keys.yaml` file should look similar to the following:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: <quayname>-quay-registry-managed-secret-keys
+  namespace: <quay-namespace>
+data:
+  CONFIG_EDITOR_PW: <redacted>
+  DATABASE_SECRET_KEY: <redacted>
+  DB_ROOT_PW: <redacted>
+  DB_URI: <redacted>
+  SECRET_KEY: <redacted>
+  SECURITY_SCANNER_V4_PSK: <redacted>
+----
++
+All information under the `data` property should remain the same.
+
+. Backup the current Quay configuration. This configuration lives in a secret pointed by the `QuayRegistry` custom resource and you can find the name by running the following command:
++
+[source,terminal]
+----
+$ oc get quayregistry <quay-registry-name> -n <quay-registry-namespace> -o json |jq -r .spec.configBundleSecret
+----
+
+. After obtaining the configuration bundle name, move its content into a new file:
++
+[source,terminal]
+----
+$ oc get secret -n <quay-registry-namespace> <config-bundle-secret-name> -o yaml > config-bundle.yaml
+----
```

Comment:
line 65 to lin 77 can be simplified as 
```
. Backup the current Quay configuration.

[source,terminal]
----
$ oc get secret -n <quay-registry-namespace>  $(oc get quayregistry <quay-registry-name> -n <quay-registry-namespace>  -o jsonpath='{.spec.configBundleSecret}') -o yaml > config-bundle.yaml
----
```
Simple steps toward delivering a more convenient customer experience.